src/components/Budget/BudgetView/BudgetViewContainer.js

```diff
@@ -170,7 +170,7 @@ const BudgetViewContainer = ({ history, location, match, mutator }) => {
         </Button>
       </IfPermission>
 
-      <IfPermission perm="finance.budgets.item.delete">
+      <IfPermission perm="ui-finance.fund-budget.delete">
```

Comment:
I don't get what is the point of such code changes, could you please provide the reason?

Reply:
to be consistent using the same permission sets in the app

Reply:
Please elaborate more on that, I don't get what are you referring to

Reply:
Following permissions guidelines, I left checking atomic permissions for creating, editing and deleting cases, and UI-permissions for lists and routes.